compose/common.yml

```diff
@@ -67,6 +67,7 @@ x-provider-server-environment: &provider-server-environment
   USE_QUEUE: ${USE_QUEUE:-false}
 
 x-sdk-environment: &sdk-environment
+  HANDLE_POLICY_IDS: ${HANDLE_POLICY_IDS:-f0ff48bbb7bbe9d59a40f1ce90e9e9d0ff5002ec48f232b49ca0fb9a}
```

Comment:
Very nice, but here have impact on _local-testnet_ as well, while KorLabs `policyId` can be used only for public networks.

Better to set this variable in `packages/cardano-services/package.json/scripts/compose:up` to correctly set it only once for all the public networks, but without impacting _local-testnet_.

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/924/commits/ec28e25ef8207de7851edbea1d00bce7aba1c79d